hasjob/templates/embedlayout.html.jinja2

```diff
@@ -0,0 +1,80 @@
+{%- extends "baseframe.html.jinja2" -%}
+
+{%- block doctypehtml -%}
+<!DOCTYPE html>
+<!--[if lt IE 7]> <html class="no-js lt-ie9 lt-ie8 lt-ie7" lang="en" {% block manifest %}{%- if g.kiosk %} manifest="{{ url_for('kiosk_manifest') }}" {%- endif %}{% endblock %}> <![endif]-->
+<!--[if IE 7]>    <html class="no-js lt-ie9 lt-ie8" lang="en" {{ self.manifest() }}> <![endif]-->
+<!--[if IE 8]>    <html class="no-js lt-ie9" lang="en" {{ self.manifest() }}> <![endif]-->
+<!--[if gt IE 8]><!--> <html lang="en" class="no-js {%- if g.user %} userlogin {%- else %} no-userlogin {%- endif %}" {{ self.manifest() }}> <!--<![endif]-->
+{%- endblock -%}
+
+{%- block titletags %}
+  <title>{% block title %}{{ title }}{% endblock %} | {% if g.board %}{{ g.board.title }}{% if g.board.not_root %} ({{ config['SITE_TITLE'] }}){% endif %}{% else %}{{ config['SITE_TITLE'] }}{% endif %}</title>
+  <meta name="DC.title" content="{{ self.title()|e }}" />
+  <meta property="og:title" content="{{ self.title()|e }}" />
+{%- endblock %}
+
+{%- block description %}{% if g.board and g.board.description %}{{ g.board.description|firstline }}{% else %}Hasjob is India’s best job board for tech startups{% endif %}{% endblock %}
+
+{% block image_src -%}
+  <link rel="image_src" href="{{ url_for('static', filename='img/logo-star.png', _external=true) }}" />
+  <meta property="og:image" content="{{ url_for('static', filename='img/logo-star.png', _external=true) }}" />
+{%- endblock %}
+
+{%- block layoutheaders %}
+  <link rel="icon" sizes="192x192" href="/static/img/hasjob-logo.svg">
+  <link rel="stylesheet" type="text/css" href='{{ asset_path('app-css') }}' />
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Fira+Sans:400,400i,700,700i|McLaren&amp;subset=latin-ext">
+  {#
+  <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=McLaren" />
+  <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Fira+Sans:400,700,400italic,700italic&subset=latin,latin-ext">
+  <link rel="stylesheet" type="text/css" href="//code.cdn.mozilla.net/fonts/fira.css">
+  -#}
+  <script>
+    if (typeof window.Hasjob === 'undefined') {
+      window.Hasjob = {};
+    }
+    // Use Hasjob.Config for server-generated responses
+    if (typeof window.Hasjob.Config === 'undefined') {
+      window.Hasjob.Config = {
+        baseURL: "/"
+      }
+    }
+  </script>
+{%- endblock %}
```

Comment:
There is too much replicated from `layout.html.jinja2` here. We'll have to find a way to abstract this out. Maybe in a separate PR.

Reply:
I extended `layout.html.jinja2` in `embedlayout.html.jinja2`

Reply:
Better now!